TDReactNativeExample/package.json

```diff
@@ -10,9 +10,11 @@
     "lint": "eslint ."
   },
   "dependencies": {
-    "react": "16.11.0",
-    "react-native": "0.62.2",
+    "@react-native-community/cli": "^7.0.3",
+    "react": "17.0.2",
+    "react-native": "0.67.3",
     "react-native-iap": "^4.5.0",
+    "react-native-macos": "^0.66.26",
```

Comment:
Wait, why do we need it?

Reply:
Ran `react-native info`, it said the lib is missing, so I just installed it :)